pkg/operator/ceph/cluster/cluster_external.go

```diff
@@ -128,6 +129,12 @@ func (c *ClusterController) configureExternalCephCluster(cluster *cluster) error
 		}
 	}
 
+	// Create cluster-wide RBD bootstrap peer token
+	_, err = controller.CreateBootstrapPeerSecret(cluster.context, cluster.ClusterInfo, &cephv1.CephCluster{ObjectMeta: metav1.ObjectMeta{Name: cluster.namespacedName.Name, Namespace: cluster.Namespace}}, cluster.ownerInfo)
```

Comment:
This will succeed in external clusters without the ceph admin user, right?

Reply:
No, it doesn't work without it 
But do we give all the cluster the admin cred as what I see the ClusterInfo is updated

https://github.com/rook/rook/blob/49e7e37341970c26dbd8c323c74bcec7beabda65/pkg/operator/ceph/cluster/cluster.go#L76

Reply:
That's just a placeholder until the operator reads the creds from the secret imported from the external cluster.

Reply:
Which makes this condition to always true
https://github.com/rook/rook/blob/49e7e37341970c26dbd8c323c74bcec7beabda65/pkg/operator/ceph/cluster/cluster_external.go#L102

Reply:
Yaa it will require more privileges to do this task